wgpu-core/src/binding_model.rs

```diff
@@ -697,7 +697,7 @@ pub enum BindError {
         s1 = if *.actual >= 2 { "s" } else { "" },
     )]
     MismatchedDynamicOffsetCount {
-        group: u8,
+        group: u32,
```

Comment:
Oi, this is, technically, a semver breaking change. 

I think we could validly argue that while it is breaking, no one is going to be looking at it, only calling Display::fmt.

Let's go ahead with it.